packages/config/src/Config.ts

```diff
@@ -92,6 +92,7 @@ export interface Config {
 
   userAgent?: string
   tag?: string
+  updateNotifier: boolean
```

Comment:
make it optional because even when it is true, in many cases updates are not checked (for instance, in CI)